src/input/label.component.ts

```diff
@@ -148,6 +174,7 @@ export class Label implements AfterContentInit, AfterViewInit {
 	@ContentChild(TextArea, { static: false }) textArea: TextArea;
 
 	@HostBinding("class.bx--form-item") labelClass = true;
+	@HostBinding("class.bx--text-input-wrapper") inputWrapperClass = true;
```

Comment:
Is this intentional? It will be assigned to the host in both text area & text input or no matter what the content is 🤔

Reply:
It's what makes the `inline` variant work... I currently have it as a viable option for textarea as well.

Reply:
`${prefix}--text-input--${size}` is set in the `ibmText` directive:
https://github.com/carbon-design-system/carbon-components-angular/blob/master/src/input/input.directive.ts#L26-L34